frontend/src/components/menuDesktop/MenuDesktop.tsx

```diff
@@ -16,51 +16,59 @@ const icons = [
   return (
     <>
       <div className={styles.logo_div}>
         <img className={styles.logo_menu} src='/assets/icons/logo-menu.svg' alt='Logo' />
         <h4>Dashhhboard</h4>
-        <img className={styles.logo_menu} src='/assets/icons/plus.svg' alt='Plus Icon' />
       </div>
       <List className={styles.list_top}>
         {['Dashboard', 'Project', 'My Task', 'Activity', 'Team', 'Message', 'Setting'].map(
           (text, index) => (
-            <ListItem key={text} disablePadding className={styles.list_item}>
-              <ListItemButton>
-                <Link
-                  className={styles.list_item_link}
-                  to={index === 0 ? '/' : `/${text.split(' ').join('').toLowerCase()}`}
-                >
-                  <ListItemIcon>
+            <Link
+              key={text}
+              className={styles.list_item_link}
+              to={index === 0 ? '/' : `/${getPath(text)}`}
+            >
+              <ListItem
+                disablePadding
+                className={`${styles.list_item} ${
+                  (location.pathname === '/' && text === 'Dashboard') ||
+                  location.pathname.includes(getPath(text))
+                    ? styles.current_path
+                    : ''
+                }`}
+              >
+                <ListItemButton>
+                  <ListItemIcon style={{ minWidth: '1.25rem', paddingRight: '1rem' }}>
                     <img
                       className={styles.icons}
                       src={`/assets/icons/${icons[index]}.svg`}
                       alt={`${text}-icon`}
                     />
                   </ListItemIcon>
-                  <ListItemText className={styles.list_name} primary={text} />
-                </Link>
-              </ListItemButton>
-            </ListItem>
+                  <div className={styles.list_name}>{text}</div>
+                </ListItemButton>
+              </ListItem>
+            </Link>
           ),
         )}
       </List>
       <List id={styles.list_bottom}>
         {['Workspace', 'Superboard'].map((text, index) => (
           <ListItem key={text} disablePadding className={index >= 7 ? 'last-two-items' : ''}>
-            <ListItemButton>
-              <Link className={styles.list_item_link} to={`/${text.toLowerCase()}`}>
-                <ListItemIcon>
-                  <img
-                    className={styles.icons}
-                    src={`/assets/icons/${icons.slice(-2)[index]}.svg`}
-                    alt={`${text}-icon`}
-                  />
-                </ListItemIcon>
-                <ListItemText className={styles.list_name} primary={text} />
-              </Link>
-            </ListItemButton>
+            {text !== 'Workspace' ? (
+              <ListItemIcon style={{ minWidth: '1.25rem', padding: '0 1rem' }}>
+                <img
+                  className={styles.icons}
+                  src={`/assets/icons/${icons.slice(-2)[index]}.svg`}
```

Comment:
similar issue to other comment. There is a clear expectation of how the data look but no comments to explain it.